tests/cbmc/include/cbmc_proof/nondet.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+/**
+ * Non-determinstic functions used in CBMC proofs
```

Comment:
Perhaps one additional sentence for anybody who is curious where these are defined –– explain that CBMC will emit nondeterministic values precisely because these functions have no definition?

Reply:
I am curious on why these functions have no definition here and if there are any consequences of using non-deterministic functions with the CBMC proofs.

Reply:
Clarified in the comment

Reply:
@ttjsu-aws is this clear now? Unconstrained values are an unusual feature of CBMC, and I want to make sure the documentation makes it clear what's going on.